tests/meson.build

```diff
@@ -82,7 +90,8 @@ if gtest_dep.found()
   unittest_plugins = executable('unittest_plugins',
     join_paths('nnstreamer_plugins', 'unittest_plugins.cpp'),
     dependencies: [nnstreamer_unittest_deps],
-    install: false
+    install: install_test,
```

Comment:
update this for `unittest_src_iio` as well? (line 103)

Reply:
Oh.. I didn't aware of that. I will check it and update it if needed. Thanks.

Reply:
Done.